adage/adage/settings.py

```diff
@@ -82,12 +82,18 @@
                     "tokenizer": "standard",
                     "filter": [
                         "standard",
-                        "word_delimiter",
+                        "adage_word_delimiter",
                         "lowercase",
                         "stop",
                         "snowball"
                     ]
                 },
+            },
+            'filter': {
+                'adage_word_delimiter': {
+                    'type': 'word_delimiter',
+                    'split_on_case_change': False
```

Comment:
Ahh... nice catch. I saw that when I started editing the page and meant to correct it. This file hasn't been touched in a while, apparently, and was "grandfathered" in before we mandated PEP8 compliance. Will fix that.

- [ ] fix string delimiter inconsistency in settings.py